policies/loading-frame-default-eager.md

```diff
@@ -0,0 +1,57 @@
+loading-frame-default-eager Policy
+===========
+
+The `loading-frame-default-eager` policy-controlled feature can be used for a main frame or a nested
+`<iframe>` to override the default behavior of the newly
+introduced `loading` [attribute](https://github.com/whatwg/html/pull/3752).
+
+What does that mean?
+------------
+With the `loading-frame-default-eager` attribute, developers could prioritize the loading of different inline frames on a web page. This however could become a cumbersome process and not quite scalable for larger
+web sites specially given that applying the attribute is origin-agnostic. The proposed policy aims to resolve
+this issue but changing a browser's default decision for `loading` behavior.
```

Comment:
> The proposed policy aims to resolve
> this issue but changing a browser's default decision

"_by_ changing a browser's..."?

Reply:
These typos also exist in https://github.com/w3c/webappsec-feature-policy/pull/295/files :(

Reply:
Done thanks. and yes it should definitely be sharing the same typos which I will tend to tomorrow.